integration-tests/src/test/java/com/datastax/oss/driver/api/core/specex/SpeculativeExecutionIT.java

```diff
@@ -343,54 +350,67 @@ private CqlSession buildSessionWithProfile(
       int profile1MaxSpeculativeExecutions,
       long profile1SpeculativeDelayMs) {
 
-    List<String> config = Lists.newArrayList();
-    config.add(String.format("basic.request.timeout = %d milliseconds", SPECULATIVE_DELAY * 10));
-    config.add("basic.request.default-idempotence = true");
-    config.add(
-        "basic.load-balancing-policy.class = com.datastax.oss.driver.api.testinfra.loadbalancing.SortingLoadBalancingPolicy");
+    ProgrammaticDriverConfigLoader.Builder builder =
```

Comment:
Example of using this with profiles.

Reply:
I see no profile ref here or below. What am I missing?